dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/configuration/wrapper/DcesDrcApiResponse.java

```diff
@@ -9,7 +9,7 @@
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
-public class CustomResponse<T> {
+public class DcesDrcApiResponse<T> {
```

Comment:
Is this response type going to be used regardless of error or not? 
If it is just error, can we put that in? Just to express more of it's purpose in name.

Reply:
DcesDrcErrorResponse or something akin. As it stands, it looks like a wrapper for any responses our api would send.